content/en/docs/concepts/storage/volumes.md

```diff
@@ -11,12 +11,12 @@ weight: 10
 
 <!-- overview -->
 
-On-disk files in a Container are ephemeral, which presents some problems for
-non-trivial applications when running in Containers.  First, when a Container
-crashes, kubelet will restart it, but the files will be lost - the
-Container starts with a clean state.  Second, when running Containers together
-in a `Pod` it is often necessary to share files between those Containers.  The
-Kubernetes `Volume` abstraction solves both of these problems.
+On-disk files in a container are ephemeral, which presents some problems for
+non-trivial applications when running in containers. First, if a container
+crashes, the files in the container are lost. The kubelet restarts the container
```

Comment:
Hi. This has been a good (yes tedious to edit and review) exercise. The page needs basic updating as
there are many minor formatting nits. The page is long (+1400 lines). I thought it is a good place
to look at the styling and usage of terms (resources, drivers, plugins, ...). I realized that is is hard to
define one rule that fits all. I chose to "try out" a few styling options to test the content.

The question about placing the kubelet in code font has come up before. If I remember correctly, it
was decided that the kubelet (refer to as "the kubelet" in text) should not be formatted as code.
If a sentence begins with a tool name such as `kubectl`, you do not have to capitalize this name.
Try and reword the sentence so that the sentence does not begin with this name.